nf_core/__main__.py

```diff
@@ -486,22 +445,16 @@ def download_pipeline(
 ):
     """
     Download a pipeline, nf-core/configs and pipeline singularity images.
-
-    Collects all files in a single archive and configures the downloaded
-    workflow to use relative paths to the configs and singularity images.
     """
-    from nf_core.pipelines.download import DownloadWorkflow
-
-    if tower:
-        log.warning("[red]The `-t` / `--tower` flag is deprecated. Please use `--platform` instead.[/]")
-
-    dl = DownloadWorkflow(
+    pipelines_download(
+        ctx,
         pipeline,
         revision,
         outdir,
         compress,
         force,
-        tower or platform,  # True if either specified
+        tower,
+        platform,
```

Comment:
how about we keep the `tower or platform` logic here and just use platform on the lower level functions?

Reply:
isn't is better to maintain it only in one place?

Reply:
yes, if I understand my proposal correctly, it would only be here in the `__main__.py` which is closest to the user and we would not care about `tower` anywhere else

Reply:
ah, okay, I see... well one is deprecated so we will not touch that more often and it still feels to me like this should be closest to the user and we shouldn't have to think about tower anywhere else...but up to you if you prefer it the current setup is also fine for me.